app/src/main/java/com/nervousfish/nervousfish/LoginActivity.java

```diff
@@ -12,58 +13,50 @@
 @SuppressWarnings("PMD")
 public class LoginActivity extends Activity {
 
-    private static final String DEMO_EMAIL = "espresso@spoon.com";
-    private static final String DEMO_PASSWORD = "lemoncake";
+    private static final String DEMO_PASSWORD = "12345";
 
-    private EditText mEmail, mPassword;
+    private EditText mPassword;
     private View mError;
 
+    private boolean inDevelopment = true;
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
         setContentView(R.layout.login);
 
-        mEmail = (EditText) findViewById(R.id.email);
         mPassword = (EditText) findViewById(R.id.password);
 
         View submitButton = findViewById(R.id.submit);
         submitButton.setOnClickListener(new View.OnClickListener() {
 
             @Override
             public void onClick(View v) {
-                validateFields();
-
-                if (mEmail.getError() == null && mPassword.getError() == null) {
+                if (mPassword.getError() == null) {
                     validateAccount();
                 }
             }
         });
     }
 
-    private void validateFields() {
-        if (!android.util.Patterns.EMAIL_ADDRESS.matcher(mEmail.getText().toString()).matches()) {
-            mEmail.setError(getString(R.string.msg_email_error));
-        } else {
-            mEmail.setError(null);
-        }
-
-        if (mPassword.getText().toString().isEmpty()) {
-            mPassword.setError(getString(R.string.msg_password_error));
-        } else {
-            mPassword.setError(null);
-        }
-    }
-
     private void validateAccount() {
         if (mError == null) {
             mError = findViewById(R.id.error);
         }
 
-        if (!mEmail.getText().toString().equals(DEMO_EMAIL) || !mPassword.getText().toString().equals(DEMO_PASSWORD)) {
+        //The code below has to be changed before deployment. For development
+        //you can continue when the password is empty.
```

Comment:
You can remove this comment if you create an issue for this, otherwise please change it to a TODO comment (`// TODO: Lorem ipsum dolor`)